app/presenters/updatable.rb

```diff
@@ -20,13 +25,21 @@ def history
     end
   end
 
-private
+  # The direction of change history isn’t guaranteed
+  # https://github.com/alphagov/govuk-content-schemas/issues/545
+  def reverse_chronological_change_history
+    change_history.sort_by { |item| DateTime.parse(item[:timestamp]) }.reverse
+  end
 
   def any_updates?
-    if (first_public_at = content_item["details"]["first_public_at"]).present?
+    if first_public_at.present?
```

Comment:
This implies that we `first_public_at` might not exist. I don't think we have an explicit test for this case? 
If we can get a situation of not having `first_public_at` we should perhaps test for it. If we never get a situation where that can happen then we can do away with the conditional here.